openbb_terminal/stocks/fundamental_analysis/fa_controller.py

```diff
@@ -385,24 +385,23 @@ def call_enterprise(self, other_args: List[str]):
             type=str,
             default=None,
         )
-        parser.add_argument(
-            "-l",
-            "--limit",
-            action="store",
-            dest="limit",
-            type=check_positive,
-            default=5,
-            help="Limit of latest years/quarters, relevant for the source FinancialModelingPrep.",
-        )
 
         parser.add_argument(
             "-s",
             "--start",
             type=valid_date,
-            default=(datetime.now() - timedelta(days=3 * 366)).strftime("%Y-%m-%d"),
+            default="1900-01-01",
```

Comment:
Hey, I had a query, does this change impact the response time or structure of the charts prepared(namely the axis markings getting skewed) based on the default start date?

Reply:
Within the model it gets converted to datetime actually. This is mostly for convenient sake. The default set here is essentially "give me all data available if no start or end date are set"il

It won't change anything how data is plotted or how data is collected since the functions itself work with strings like these.

Reply:
Think the question was asking about the chart performance with lots of data loaded.  For candles, there is a lot of data shown, so starting at 1900 (or however far back we can go) would make the window react slower and take up more memory.

For line charts like this, there will be a non-noticable difference between a couple thousand extra data points.

Reply:
Ah yes and note that its annual data.